# ThetaStar x RotationShimRPP: changes against /opt/ros/jazzy/share/nav2_bringup/params/nav2_params.yaml
# written by parity/scripts/generate_matrix_configs.py; 3 key(s) changed
# planner_server.GridBased.plugin: 'nav2_navfn_planner::NavfnPlanner' -> 'nav2_theta_star_planner::ThetaStarPlanner'  (plugin name)
# controller_server.FollowPath.plugin: 'nav2_mppi_controller::MPPIController' -> 'nav2_rotation_shim_controller::RotationShimController'  (plugin name)
# controller_server.FollowPath.primary_controller: '<not set>' -> 'nav2_regulated_pure_pursuit_controller::RegulatedPurePursuitController'  (required: the controller the shim hands over to)
--- installed/nav2_params.yaml
+++ ThetaStar_RotationShimRPP/params.yaml
@@ -196,7 +196,8 @@
       iteration_count: 1
       model_dt: 0.05
       motion_model: DiffDrive
-      plugin: nav2_mppi_controller::MPPIController
+      plugin: nav2_rotation_shim_controller::RotationShimController
+      primary_controller: nav2_regulated_pure_pursuit_controller::RegulatedPurePursuitController
       prune_distance: 1.7
       regenerate_noises: true
       temperature: 0.3
@@ -375,7 +376,7 @@
   ros__parameters:
     GridBased:
       allow_unknown: true
-      plugin: nav2_navfn_planner::NavfnPlanner
+      plugin: nav2_theta_star_planner::ThetaStarPlanner
       tolerance: 0.5
       use_astar: false
     costmap_update_timeout: 1.0
